web-api/compose.db.yml

```diff
@@ -36,7 +36,7 @@ services:
 
   tika:
     image: apache/tika:2.8.0.0-full
-    mem_limit: 256m
+    mem_limit: 512m
```

Comment:
This seems somewhat unrelated to this PR, I'm not sure if this mem limit is just for convenience when locally developing or could have some importance in some context :thinking: 

I guess it should be fine.

Reply:
Tika wouldn't start up on a mac, so you have to run the tests at least twice. Increasing the limit helped.